core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -410,6 +405,25 @@ def interface():
                         help='Path to Data file')
     parser.add_argument('-id', '--study_ids', type=str, required=False,
                         help='Cancer Study IDs for `remove-study` command, comma separated')
```

Comment:
this command only applies to remove-study command so i would suggest to only add this to the remove_study subparser